apps/hub-app/src/components/Profile.tsx

```diff
@@ -45,55 +38,20 @@ const AvatarLg = styled(Avatar)`
 `;
 
 const Profile = () => {
-  const [profile, setProfile] = useState({
-    image: '',
-    name: '',
-    description: '',
-    emoji: '',
-    background: '',
-  });
+  const { profile } = useHausConnect();
   const handleClick = (e: MouseEvent<HTMLDivElement>) => {
     e.preventDefault();
     navigator.clipboard.writeText(`stub.eth`);
   };
 
-  const getProfile = async () => {
-    const haus = Haus.create(networks);
-    try {
-      const profile = await haus.profile.get(
-        '0xEAC5F0d4A9a45E1f9FdD0e7e2882e9f60E301156'
-      );
-      if (profile) {
-        setProfile({
-          name: profile.name || '',
-          image: profile.image || '',
-          description: profile.description || '',
-          emoji: profile.emoji || '',
-          background: profile.image || '',
-        });
-      }
-    } catch (err) {
-      console.error(err);
-    }
-  };
-
-  useEffect(() => {
-    getProfile();
-  }, []);
-
   return (
     <ProfileContainer>
-      <AvatarLg
-        size="lg"
-        src={profile.image}
-        alt="profile image"
-        fallback={<StyledGhost />}
-      />
+      <AvatarLg src={profile?.image || ''} alt="profile image" />
       <div>
         <NameContainer>
-          <H5>{profile.name}</H5>
+          <H5>{profile?.name || ''}</H5>
```

Comment:
Do you need to render an empty H5 if name doesn't exist?
Something like
```suggestion
         {profile?.name && <H5>{profile?.name}</H5>}
```

Reply:
Happy to change it is there any visual difference? Is it mainly to remove markup clutter

Reply:
It's  two fold.
You'll get an "Empty Heading Tag" accessibility checkers. A screen reader will alert a user that there is an empty tag, h5 in this case and stop. The user has to progress to the next element.
Second is to keep the markup cleaner.

If you get an error you can covert to a boolean
`{!!item.mystring && <H5>...`
Or use a ternary operator and return null.
```
{item.mystring ?
 <H5>...
  :
  null
}
```